### PR TITLE
Adding the ability to create a container with analytical storage turned on.

### PR DESCRIPTION
**Pull request by toswedlu**
Just adding another parameter to 'create_container' on the database proxy object such that the TTL for analytical storage can be set, effectively turning it on.

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - Added deprecation warning for "lazy" indexing mode. The backend no longer allows creating containers with this mode and will set them to consistent instead.
 - Fix for bug where options headers were not added to upsert_item function. Issue #11791 - thank you @aalapatirvbd.
 - Fixed error raised when a non string ID is used in an item. It now raises TypeError rather than AttributeError. Issue #11793 - thank you @Rabbit994.
+- Added the ability to set the analytical storage TTL we creating a new container.
```
Comment (southpolesteve) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:6]:
```suggestion
- Added the ability to set the analytical storage TTL when creating a new container.
```

**Comment by annatisch** [comment]
Hi @toswedlu - these new tests appear to be failing when run against a live endpoint. Could you take a look?

**Pull request merged**